### PR TITLE
Add support for openscap plugins

### PR DESCRIPTION
**Pull request by ares**
depends on https://github.com/theforeman/puppet-foreman/pull/273 and https://github.com/theforeman/puppet-foreman_proxy/pull/141

### CODE REVIEW

**Comment by mmoll** [comment]
the both PRs are merged now, however I don't have a RH to test...

**Comment by ares** [comment]
do not review/merge this yet, there were changes in those PRs so I have to update this one and test, sorry I should have posted a warning here

File: config/foreman-installer.yaml [hunk 1]
```diff
@@ -58,6 +58,9 @@
   :foreman::plugin::docker:
     :dir_name: foreman
     :manifest_name: plugin/docker
+  :foreman::plugin::openscap:
```
Comment (domcleal) [inline config/foreman-installer.yaml:61]: may also need a params_name?

File: Puppetfile [hunk 2]
```diff
@@ -12,3 +12,6 @@ mod 'theforeman/tftp',          :git => 'https://github.com/theforeman/puppet-tf
 mod 'theforeman/foreman',       :git => 'https://github.com/theforeman/puppet-foreman'
 mod 'theforeman/foreman_proxy', :git => 'https://github.com/theforeman/puppet-foreman_proxy'
 mod 'theforeman/puppet',        :git => 'https://github.com/theforeman/puppet-puppet'
+
+# Plugin dependencies
+mod 'OpenSCAP/foreman_scap_client', :git => 'https://github.com/OpenSCAP/puppet-foreman_scap_client'
```
Comment (domcleal) [inline Puppetfile]: obsolete?

**Comment by ares** [comment]
seems to work for me now, foreman won't start on el7 but I believe by @domcleal already (ffi/selinux) and other engine errors should be fixed in new release

**Comment by domcleal** [comment]
Merged as a3eafaa, thanks @ares!

**Pull request closed without merging**